conjure_oxide/tests/integration/basic/abs/01-simple/input.eprime

```diff
@@ -0,0 +1,5 @@
+language ESSENCE' 1.0
+
+find x,y: int(-5..5)
```

Comment:
change this to something other that -5..5 and we would be testing the bound flipping?